Detectors/TRD/calibration/src/CalibratorVdExB.cxx

```diff
@@ -14,49 +14,48 @@
 /// \author Ole Schmidt
 
 #include "TRDCalibration/CalibratorVdExB.h"
-#include "Fit/Fitter.h"
+#include "DataFormatsTRD/Constants.h"
 #include "TStopwatch.h"
 #include "CCDB/CcdbApi.h"
 #include "CCDB/BasicCCDBManager.h"
-#include <string>
-#include <map>
-#include <memory>
 #include "CommonUtils/NameConf.h"
 #include "CommonUtils/MemFileHelper.h"
 #include <TFile.h>
 #include <TTree.h>
 
+#include <string>
+#include <map>
+#include <memory>
+#include <ctime>
+
 using namespace o2::trd::constants;
 
 namespace o2::trd
 {
 
 double FitFunctor::calculateDeltaAlphaSim(double vdFit, double laFit, double impactAng) const
 {
-
-  double trdAnodePlane = .0335; // distance of the TRD anode plane from the drift cathodes in [m]
```

Comment:
Actually, I think this is a typo. Should be 3.5 cm and this can be taken from the geometry: `o2::trd::GeometryBase::camHght()/(2.f * 100.f)` the 100 to go from cm to m